tensorflow/compiler/xla/tests/exhaustive_op_test_utils.h

```diff
@@ -30,6 +30,26 @@ limitations under the License.
 namespace xla {
 using Eigen::half;
 
+namespace int_type {
```

Comment:
Maybe put this in a `test_util` namespace?

Reply:
sure thing